lib/honeycomb/propagation/aws.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require "base64"
+require "json"
+require "uri"
```

Comment:
Are any of these `require` things used in this file?

Reply:
Nope, thanks for catching. Removing!